service/storage_proxy.cc

```diff
@@ -377,18 +377,22 @@ class storage_proxy::remote {
             tracing::trace(trace_state_ptr, "Message received from /{}", src_addr.addr);
         }
 
-        return do_with(std::vector<frozen_mutation_and_schema>(),
-                       [this, cl, src_addr, timeout = *t, fms = std::move(fms), trace_state_ptr = std::move(trace_state_ptr)] (std::vector<frozen_mutation_and_schema>& mutations) mutable {
-            return parallel_for_each(std::move(fms), [this, &mutations, src_addr] (frozen_mutation& fm) {
+        std::vector<frozen_mutation_and_schema> mutations;
+        auto timeout = *t;
+        {
```

Comment:
Why the scope?

Reply:
I kept the indentation, so added scopes to avoid confusing the IDE. They are removed in the last patch.